### DISCUSSION TITLE
gutters

### DISCUSSION THREAD

**Discussion by 209** [quasarframework/quasar#11219] [General - Components / Directives / etc] [answered]
I don't understand, it's correctly gutter's behavior? 
https://jsfiddle.net/1v2gqdfz/28/

<img width="647" alt="Снимок экрана 2021-11-04 в 21 34 05" src="https://user-images.githubusercontent.com/4629854/140399409-bb3d7a16-acb6-43e4-87e5-7d07eaeccff6.png">

**Comment by pdanpdan** [comment, marked as answer]
https://quasar.dev/layout/grid/gutter#pros-cons-and-how-to-workaround-problems-q-gutter-size-vs-q-col-gutter-size

**Reply by 209** [reply]
thanks!
